map-utils.js

```diff
@@ -1,126 +1,139 @@
+  var endLng = degreesToRadians(posB.lng || posB.longitude);
+  var partialResult = Math.sin(startLat) * Math.sin(endLat)
+  + Math.cos(startLat) * Math.cos(endLat) * Math.cos(startLng - endLng);
+  var distance = Math.acos(Math.min(partialResult, 1)) * EARTH_RADIUS;
+
+  return Number(distance.toFixed(3));
+};
+var getRadiusLatLng = function (latLng, radius){
+  var startLat = degreesToRadians(latLng.lat);
+  var startLng = degreesToRadians(latLng.lng);
+  var dist = parseFloat(radius) / (EARTH_RADIUS * 1000);
+  var destAng = degreesToRadians(0);
+  var destLat = Math.asin(Math.sin(startLat) * Math.cos(dist)
+   + Math.cos(startLat) * Math.sin(dist) * Math.cos(destAng));
+  var destLng = ((startLng + Math.atan2(Math.sin(destAng) * Math.sin(dist)
+                  * Math.cos(startLat),
+                  Math.cos(dist) - Math.sin(startLat) * Math.sin(destLat)))
+                  * 180) / Math.PI;
+  destLat = (destLat * 180) / Math.PI;
+
+  return { lat: destLat, lng: destLng };
+};
+var getPointFromLatLng = function (latLng, opts){
+  opts = opts || { asPlainObject: false };
+  var siny = Math.max(Math.sin(degreesToRadians(latLng.lat)), -0.9999);
+  var x = TILE_SIZE * (0.5 + latLng.lng / 360);
+  var y = TILE_SIZE * (0.5 - Math.log((1 + siny) / (1 - siny)) /
+                              (4 * Math.PI));
+  if (opts.asPlainObject){
+    return { x: x, y: y };
+  }
+
+  return new google.maps.Point(x, y);
+};
+var getLatLngFromPoint = function (point){
+  var lng = (point.x - tilePixelCenter) / pixelsPerLngDegree;
+  var latRadians = (point.y - tilePixelCenter) / -pixelsPerLngRadian;
+  var lat = radiansToDegrees(2 * Math.atan(Math.exp(latRadians)) - Math.PI / 2);
+
+  return { lat: lat, lng: lng };
+};
+var getFormattedPlaceName = function (placeObj){
+  if (!placeObj.formatted_address){
+    return null;
+  }
+  var htmlAddress = placeObj.adr_address;
+  var firstPart = htmlAddress.substr(0, htmlAddress.indexOf('<'));
+  var addressContainer = window.document.createElement('span');
+  addressContainer.innerHTML = htmlAddress;
+  var childNodes = Array.prototype.slice.call(addressContainer.children, 0);
+  if (childNodes.length === 0) {
+    return placeObj.formatted_address;
+  }
+
+  return firstPart + childNodes
+  .filter(function (node) { return node.className !== 'postal-code';})
+  .map(function (node) { return node.textContent;}).join(', ');
+
+};
+var getAdjustedMapCenter = function (options){
+  var mapCenter = options.mapCenter;
+  var mapZoom = options.mapZoom;
+  var offset = options.offset;
+  var reverse = options.reverse;
+  var directionalOffset = reverse ? offset * -1 : offset;
+  var z = Math.pow(2,mapZoom);
+  var result;
+  var point = getPointFromLatLng({
+    lat: result(mapCenter, 'lat'),
+    lng: result(mapCenter, 'lng')
+  }, { asPlainObject: true });
+  var adjustedMapCenter = getLatLngFromPoint({
+    x: point.x,
+    y: point.y + (directionalOffset / 2) / z
+  });
+  var round;
```

Comment:
This one is tricky. Since round is a method, it's technically defined elsewhere (should be treated as a global) and this line will be ignored in code. Since I removed the `require('lodash')` lines that imported this, you wouldn't know, so this is more of just an fyi.